microcosm_pubsub/dispatcher.py

```diff
@@ -75,7 +83,15 @@ def configure_sqs_message_dispatcher(graph):
     except NotBoundError:
         sqs_message_handlers = graph.config.sqs_message_dispatcher.mappings
 
+    try:
```

Comment:
I removed the `log_with_context` flag in favor of having a noop default that we can always use. User can have more fine-grained control by binding the key `sqs_message_context` with either the default function from this repo or their own custom context.